src/sentry/api/bases/user.py

```diff
@@ -26,8 +27,13 @@ def has_object_permission(self, request: Request, view, user: User | RpcUser | N
             return True
         if request.auth:
             return False
+
         if is_active_superuser(request):
-            return True
+            permissions = access_service.get_permissions_for_user(request.user.id)
```

Comment:
Might be good to add a comment here saying what permissions these are like:
https://github.com/getsentry/sentry/blob/827d488b965425a4c4d5d9b1636e8e978fa8686e/src/sentry/services/hybrid_cloud/access/service.py#L121-L122
Scopes vs permission classes vs user/admin permissions are confusing

Reply:
overloaded words 😵‍💫